File: app/src/main/java/me/kirimin/mitsumine/ui/fragment/ReadFeedFragment.java
package me.kirimin.mitsumine.ui.fragment;

import me.kirimin.mitsumine.db.FeedDAO;
import me.kirimin.mitsumine.model.Feed;

public class ReadFeedFragment extends AbstractFeedFragment {

    @Override
    void requestFeed() {
        setFeed(FeedDAO.findByType(Feed.TYPE_READ));
    }

    @Override
    boolean isUseReadLater() {
        return true;
    }

    @Override
    boolean isUseRead() {
        return false;
    }
}

File: app/src/main/java/me/kirimin/mitsumine/ui/fragment/ReadFeedFragment.kt
package me.kirimin.mitsumine.ui.fragment

import me.kirimin.mitsumine.db.FeedDAO
import me.kirimin.mitsumine.model.Feed

public class ReadFeedFragment : AbstractFeedFragment() {

    override fun requestFeed() {
        setFeed(FeedDAO.findByType(Feed.TYPE_READ))
        dismissRefreshing()
    }

    override fun isUseReadLater(): Boolean {
        return true
    }

    override fun isUseRead(): Boolean {
        return false
    }
}
